src/Shouldly.Tests/ConventionTests/ApprovePublicApi.ShouldlyApi.approved.cs

```diff
@@ -206,6 +206,8 @@ public static void ShouldBeOfTypes<T>(this System.Collections.Generic.IEnumerabl
         public static void ShouldBeOfTypes<T>(this System.Collections.Generic.IEnumerable<T> actual, System.Type[] expected, string? customMessage) { }
         public static void ShouldBeSubsetOf<T>(this System.Collections.Generic.IEnumerable<T> actual, System.Collections.Generic.IEnumerable<T> expected, string? customMessage = null) { }
         public static void ShouldBeSubsetOf<T>(this System.Collections.Generic.IEnumerable<T> actual, System.Collections.Generic.IEnumerable<T> expected, System.Collections.Generic.IEqualityComparer<T> comparer, string? customMessage = null) { }
+        public static void ShouldBeSupersetOf<T>(this System.Collections.Generic.IEnumerable<T> actual, System.Collections.Generic.IEnumerable<T> expected, string? customMessage = null) { }
```

Comment:
```suggestion
        public static T ShouldBeSupersetOf<T, U>(this T actual, System.Collections.Generic.IEnumerable<U> expected, string? customMessage = null) where T : System.Collections.Generic.IEnumerable<T> { }
```

I suggest such signature for both methods to allow chaining.

Reply:
I suspected such answer.